jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -1424,7 +1427,7 @@
     (let (avs)
       (setq avs
             (list
-              (send *baxter* :approach-from-downside arm)
+              (send *baxter* :midpose-hold-opposite-hand-object-downside arm)
```

Comment:
For example, we have `fold-pose-lower`, a predefined pose, but it can't be used for the `:hold-opposite-hand-object` because of collision with robot itself or tote, right?